.pre-commit-config.yaml

```diff
@@ -0,0 +1,60 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v3.4.0
+    hooks:
+      - id: trailing-whitespace
+      - id: check-added-large-files
+      - id: check-json
+      - id: check-ast
+      - id: check-merge-conflict
+      - id: check-toml
+      - id: check-yaml
+        args: [--unsafe]
+      - id: debug-statements
+      - id: detect-private-key
+      - id: end-of-file-fixer
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.790
+    hooks:
+      - id: mypy
+  - repo: https://github.com/ambv/black
+    rev: 20.8b1
+    hooks:
+      - id: black
+        name: Run black formatter
+        language_version: python3.8
+  - repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+      - id: flake8
+        name: Run flake8 linter
+        additional_dependencies: ["flake8-bugbear==20.1.4", "pep8-naming==0.11.1"]
+  - repo: https://github.com/timothycrosley/isort
```

Comment:
This sorts the imports. [By default it](https://github.com/PyCQA/isort#controlling-how-isort-sections-from-imports) applies the three sections and puts `import ...` above `from .. import ...`